packages/assets-controllers/src/NftController.ts

```diff
@@ -1395,24 +1439,24 @@ export class NftController extends BaseControllerV1<NftConfig, NftState> {
    */
   async updateNftMetadata({
     nfts,
-    userAddress = this.config.selectedAddress,
+    userAddress = this.#config.selectedAddress,
     networkClientId,
   }: {
     nfts: Nft[];
     userAddress?: string;
     networkClientId?: NetworkClientId;
   }) {
-    const chainId = this.getCorrectChainId({ networkClientId });
+    const chainId = this.#getCorrectChainId({ networkClientId });
 
     const nftsWithChecksumAdr = nfts.map((nft) => {
       return {
         ...nft,
         address: toChecksumHexAddress(nft.address),
       };
     });
-    const nftMetadataResults = await Promise.allSettled(
+    const nftMetadataResults = await Promise.all(
```

Comment:
Is there a reason we're moving away from `allSettled` here?

Reply:
yes because the promise we are looping is `#getNftInformation` which uses safelyExecute which will never fail

Reply:
Got it, guess `#getNftInformation` has always been returning fulfilled promises only, in which case there's no point in using `allSettled` over `all`. Good call!